Makefile

```diff
@@ -20,10 +20,17 @@ test-clean:
 test:
 	go test -v ./cmd/... ./pkg/...
```

Comment:
How about
```
GOTEST ?= "go test"

test:
	$(GOTEST) -v ./cmd/... ./pkg/...
```

Then you can you can run it like this: `GOTEST="gotestsum" make test"

It feels weird because this doesn't do anything different other than replacing go test with an external exec

Reply:
The -v must be inside the GOTEST=.

The downside of this approach is it means I need to remember the syntax. I guess the makefile can have a comment (or a target that sets GOTEST?)